test/routesItems.test.js

```diff
@@ -0,0 +1,181 @@
+var server = require('../index');
+var test = require('tape');
+var task;
+
+server.inject('/tasks', (res) => {
+  task = res.result.tasks[res.result.tasks.length - 1];
+  var item;
+
+  // Return a item randomly
+  test(`POST /tasks/${task.idtask}/items`, function(t) {
+    var options = {
+      method: 'POST',
+      url: `/tasks/${task.idtask}/items`,
+      payload: {
+        user: 'rub21',
+        editor: 'josm'
+      }
+    };
+    server.inject(options, (res) => {
+      item = res.result;
+      t.equal(res.statusCode, 200, 'HTTP 200 OK');
+      t.equal(res.result.type, 'Feature', 'Feature OK');
+      t.equal(res.result.properties._tofix.length, 1, 'Num Editions 1 OK');
+      t.end();
+    });
+  });
+
+  var keys;
+  var key;
+  // Group of items
```

Comment:
We can get into that discussion later, in the meantime expanding on the comments to make this clear would be a good idea.